semanticdb-kotlinc/src/test/kotlin/com/sourcegraph/semanticdb_kotlinc/test/AnalyzerTest.kt

```diff
@@ -536,4 +556,73 @@ class AnalyzerTest {
 
         result.exitCode shouldBe KotlinCompilation.ExitCode.OK
     }
+
+    @Test
+    fun documentation(@TempDir path: Path) {
+        val document =
+            compileSemanticdb(
+                path,
+                """
+               package sample
+               import java.io.Serializable
+               abstract class DocstringSuperclass
+
+               /** Example class docstring. */
+               class Docstrings: DocstringSuperclass(), Serializable
+               
+               /** Example method docstring. */
+               inline fun docstrings(msg: String): Int { return msg.length }
+        """.trimIndent())
+        val obtainedSymbols =
+            document
+                .symbolsList
+                .map { "----------\n${it.symbol}\n${it.documentation.message}" }
+                .joinToString("\n")
+        val expectedSymbols =
+            """
+----------
+sample/DocstringSuperclass#
+```kt
+class DocstringSuperclass
+```
+----------
+sample/DocstringSuperclass#`<init>`().
+```kt
+constructor DocstringSuperclass()
+```
+----------
+sample/Docstrings#
+```kt
+class Docstrings : sample.DocstringSuperclass, java.io.Serializable
+```
+
+----
+
+Example class docstring
+----------
+sample/Docstrings#`<init>`().
+```kt
+constructor Docstrings()
+```
+
+----
+
+Example class docstring
+----------
+sample/TestKt#docstrings().
+```kt
+inline fun docstrings(msg: kotlin.String): kotlin.Int
+```
+
+----
+
+Example method docstring
+----------
+sample/TestKt#docstrings().(msg)
+```kt
+value-parameter msg: kotlin.String
+```
+""".trim()
+        assertEquals(expectedSymbols, obtainedSymbols)
+    }
```

Comment:
If its nicer for you, you can indent with leading `|` and do `trimMargin()` instead of `trim()`

Reply:
I did that at first but ktfmt removed the `|` from empty lines, which was annoying

Reply:
(still like ktfmt haha)

Reply:
Wow thats...annoying :thonk: